src/models/token.js

```diff
@@ -0,0 +1,15 @@
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../utils/db.js');
+const User = require('./user.js');
+
+const Token = sequelize.define('Token', {
+  refreshToken: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+});
+
+Token.belongsTo(User);
+User.hasOne(Token);
```

Comment:
Ensure that the associations between `Token` and `User` are correctly set up in the database. The `belongsTo` and `hasOne` associations imply a one-to-one relationship, which should be reflected in the database schema.